src/html/Quoting.cc

```diff
@@ -9,50 +9,48 @@
 #include "squid.h"
 #include "html/Quoting.h"
 
-/*
- *  HTML defines these characters as special entities that should be quoted.
- */
-static struct {
-    unsigned char code;
-    const char *quote;
-} htmlstandardentities[] =
+#include <array>
+#include <cstring>
+
 
+static const auto& MakeEscapeSequences()
```

Comment:
Please follow Squid Style (and run formatting to fix a couple of other [problems](https://github.com/squid-cache/squid/actions/runs/6630677868/job/18012571733?pr=1513)):

```suggestion
static const auto &
MakeEscapeSequences()
```

Reply:
done